src/sentry/db/models/fields/node.py

```diff
@@ -170,6 +170,9 @@ def __init__(self, *args, **kwargs):
         self.ref_version = kwargs.pop("ref_version", None)
         self.wrapper = kwargs.pop("wrapper", None)
         self.id_func = kwargs.pop("id_func", lambda: b64encode(uuid4().bytes))
+        # We only automatically save data to nodestore for RawEvent
+        # Event performs saving node data as a separate step
+        self.skip_nodestore_save = kwargs.pop("skip_nodestore_save", False)
```

Comment:
add a feature check here so we can revert quickly in case of issues ?

Reply:
Good idea, added this. Defaults to on.